File: JSBASIC/68_leap_year.js
// 68	68_leap_year.js	Check whether a given year is a leap year or not.

let year = 2024;
console.log("Year:", year);
if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
    console.log(year, "is a Leap Year.");
} else {
    console.log(year, "is NOT a Leap Year.");
}

File: JavaBasic/68_leap_year.java
// 68	68_leap_year.java	Check whether a given year is a leap year or not.
public class _68_leap_year {
    public static void main(String[] args) {
        int year = 2024;
        System.out.println("Year: " + year);
        if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0)
            System.out.println(year + " is a Leap Year.");
        else
            System.out.println(year + " is NOT a Leap Year.");
    }
}
